Source/WebCore/rendering/RenderBlock.cpp

```diff
@@ -953,6 +953,12 @@ LayoutUnit RenderBlock::marginIntrinsicLogicalWidthForChild(RenderBox& child) co
 
 void RenderBlock::layoutPositionedObject(RenderBox& r, bool relayoutChildren, bool fixedPositionObjectsOnly)
 {
+    if (isSkippedContentRoot()) {
```

Comment:
I should have asked this earlier, but is this correct when `ContentVisibilityForceLayout` is on ? Or is this unreachable/irrelevant code when `ContentVisibilityForceLayout` is on?